tasks/main.yml

```diff
@@ -134,7 +156,9 @@
     backup: true
     mode: 0644
   notify: restart phc2sys
-  when: timesync_mode == 2 and timesync_mode2_hwts
+  when:
+    - timesync_mode == 2
+    - timesync_mode2_hwts
```

Comment:
Is this really guaranteed to be identical? I am asking because Ansible handles bare variables specially. See ansible/ansible#39414 ansible/ansible#51030 ansible/ansible#60943